### PR TITLE
Used vuepress tip cards instead of Markdown blockquotes where appropiate

### PR DESCRIPTION
**Pull request by itsWindows11**
Stuff worked on here:

- Switched to vuepress notes instead of blockquotes where appropriate.
- Fixed images in the case of selfhosting the website.
- Moved "What do you need to back up" above 3-2-1 rule.

### CODE REVIEW

**Review by heftymouse — APPROVED** [review]

File: wiki/backup.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ File History is a tool which automatically makes backups of your files. You can
 
 You can use System Image Backup to back up your system image. A system image is basically an image of the current state of the operating system stored in a file. You can use these system images to revert the system to a previous state.
 
-![System Image Backup option](img/backup/system-image-backup.png)
+![System Image Backup option](./img/backup/system-image-backup.png)
 
 - Search "File History" and then select it.
 - On the Control Panel sidebar, click on "System Image Backup".
-![](img/backup/create-a-system-image.png)
+![](./img/backup/create-a-system-image.png)
```
Comment (Erisa) [inline wiki/backup.md:43]: Not just this one but, what is the purpose of this change? It's not mentioned in the PR body.

File: wiki/windows-feature-store.md [hunk 2]
```diff
@@ -32,35 +32,35 @@ Remember, not all feature IDs will have any noticeable change, or any change at
...
+| 37010913                                  | Move search button to left                                             | 21536                 |
+| 37969115                                  | Show the desktop search bar                                            | 25120                 |
+| 40887771                                  | Move new search box next to widgets                                    | 25217                 |
+| 39072097                                  | New taskbar loading animation                                          | 25179 / 22622.575     |
+| 39072097                                  | Task Manager on Taskbar Conext Menu                                    | 25211                 |
+| 38764045                                  | New system tray                                                        | 25211                 |
+| 34878152                                  | Animated Settings navigation panel icons                               | 25197                 |
+| 26008830                                  | Tablet-optimized Taskbar                                               | 25197                 |
```
Comment (Erisa) [inline wiki/windows-feature-store.md:48]: A previous PR changed this from Taskbar to taskbar and now yours changes it back - can we get someone to check which is correct? >_>

**Review by Erisa — CHANGES_REQUESTED** [review]
In future please mention in the PR body if you modify things outside of whats mentioned in the title, it makes it easier to review. Even better, you could separate the PRs so they can be reviewed individually.

File: wiki/windows-feature-store.md (hunk 2 above)
Comment (itsWindows11) [reply wiki/windows-feature-store.md:48]: Probably accidental thing as a result of fixing the merge conflict. Will work on it.

File: wiki/backup.md (hunk 1 above)
Comment (itsWindows11) [reply wiki/backup.md:43]: The message of the commit that adds it describes why.

Comment (Erisa) [reply wiki/backup.md:43]: Okay so can you point that out? It's all reviewed as one PR and it's difficult to drill down into each commiit when there are many

Comment (itsWindows11) [reply wiki/backup.md:43]: Alright done, I'd suggest adding a template for issues and PRs to make stuff like this easier to write.

Comment (Erisa) [reply wiki/backup.md:43]: You lost me, what part of them breaks if one is selfhosting the site? I've built this thing before and it's worked the same, is there any specific environment that it breaks under?

**Comment by itsWindows11** [comment]
Oh crap, @kerimgrande messed up my PR again. 🤦 I guess you were right about separating this to another PR actually.

File: wiki/backup.md (hunk 1 above)
Comment (Erisa) [reply wiki/backup.md:43]: Thanks

**Comment by Erisa** [comment]
Yeah those kinds of formatting changes can be nasty for conflicts on areas that are frequently edited

**Comment by kerimgrande** [comment]
not me being accused??

**Comment by itsWindows11** [comment]
@Erisa It's the same, I'll edit that when I push the changes.
image

File: wiki/windows-feature-store.md (hunk 2 above)
Comment (kerimgrande) [reply wiki/windows-feature-store.md:48]: @Erisa reminds me; i changed it from "Taskbar" to "taskbar" was that wrong? kinda didn't make sense to me that the t was capitalized

Comment (itsWindows11) [reply wiki/windows-feature-store.md]: @Erisa Which one are you referring to exactly now? Just realized and I'm confused.

File: wiki/backup.md (hunk 1 above)
Comment (itsWindows11) [reply wiki/backup.md:43]: > You lost me, what part of them breaks if one is selfhosting the site? I've built this thing before and it's worked the same, is there any specific environment that it breaks under?

Basically every image doesn't load and results in a 404 error on my end, adding `./` fixed it. I thought this was an issue with the vuepress tips.

Comment (Erisa) [reply wiki/backup.md:43]: Oh sure, let me have a quick look 😅 might be the tips indeed

Comment (itsWindows11) [reply wiki/backup.md:43]: Weird tho, last time I tried it works on the deploy preview built on Netlify. Might be a localhost issue? I was able to reproduce that issue on `main` before iirc.

Comment (Erisa) [reply wiki/backup.md:43]: Nah you're right it happens when running locally, I think Vuepress made a change regarding asset handling, so I'll take the changes for this. ty

**Review by Erisa — APPROVED** [review]
